### DISCUSSION TITLE
Where Can I get the Packer Temporary Template?

### DISCUSSION THREAD

**Discussion by kmteam** [actions/virtual-environments#3033] [Image configuration]
I'm building the windows2019 image from the instructions [here](https://github.com/actions/virtual-environments/blob/main/help/CreateImageAndAzureResources.md)

After the packer image creation I restarted the computer so I lost the output info... now In the next steps I found that I need the TemplateFilePath, which was printed in the packer output.

Is there any way to recover it?

Thanks!

**Comment by miketimofeev** [comment]
Hi @kmteam if you haven't configured packer logs then I'm afraid the output can't be retrieved. However, you can find this json file in the storage account that was used to create the VHD image ( `"storage_account": "{{user `storage_account`}}"` section in windows2019.json).
The sample output from the packer log is:
`TemplateUri: https://<storage_account_name>.blob.core.windows.net/system/Microsoft.Compute/Images/images/<template name>.json`

**Reply by kmteam** [reply]
Thanks for all! I've finally found it by examining the storage account, anyway I think documentation should clarify that you have to store the path in a safe place as you will need later!

**Reply by miketimofeev** [reply]
@kmteam feel free to contribute 😃
